### PR TITLE
Add arm64 support

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

This adds arm64 support so it works better on my mbp m1 max :)

## Pull Request Checklist

Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch
- [x] If you are adding a dependency, please explain how it was chosen
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
- [x] If there is an issue associated with your Pull Request, align your PR
  title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -225,25 +233,27 @@ def build(_c, debug=False):
...
 
-        LOG.info("Building %s...", first_image)
+        LOG.info("Building %s (platform %s)...", first_image, PLATFORM)
         try:
             image = CLIENT.images.build(
                 path=str(CWD),
                 target=target,
                 rm=True,
                 tag=first_image,
+                platform=PLATFORM,
```
Comment (JonZeolla) [inline tasks.py:243]: This does not seem to be effective.  I believe it is related to https://github.com/docker/docker-py/issues/2822

**Review by JonZeolla — COMMENTED** [review]
This is blocked on `docker-py` adding platform support in their builds.

File: tasks.py (hunk 1 above)
Comment (JonZeolla) [reply tasks.py:240]: This was merged and a beta release was cut.  Just waiting on GA now I think, will test as soon as I have time

**Comment by JonZeolla** [comment]
Rebased on main, fixed linter issues.  Blocked on arm64-based hosted github runners

**Comment by JonZeolla** [comment]
Let's just merge this so we can build arm containers locally at least

**Review by digeseiso — APPROVED** [review]

**Pull request merged**